bolt-simple-taproot.md

```diff
@@ -821,15 +831,14 @@ An offered HTLC has the following form:
     * `htlc_script_root = tapscript_root([htlc_timeout, htlc_success])`
     * `htlc_timeout`:
         ```
-        <local_htlcpubkey> OP_CHECKSIGADD <remote_htlcpubkey> OP_CHECKSIGADD 2 OP_EQUALVERIFY
-        1 OP_CHECKSEQUENCEVERIFY
+        <local_htlcpubkey> OP_CHECKSIGVERIFY
+        <remote_htlcpubkey> OP_CHECKSIG
```

Comment:
This isn't correct, the sequence is still need here as we want to make sure all spends need to wait a block.